lib/taskana-core/src/main/java/pro/taskana/ClassificationSummary.java

```diff
@@ -0,0 +1,59 @@
+package pro.taskana;
```

Comment:
Is it really required to have interface + implementation for summaries?
TaskSummary and WorkbasketSummary are simple Java Beans in the pro.taskana.model folder.
What is the reason for I/F + Impl for ClassificationSummary?

Reply:
All model-classes we´ve got implements an interface for the user-api. So all summaries should implement an interface, too - i´ve asked Holger before i´ve done it this way.